### ISSUE TITLE
CONFIG_LOGIWHEELS_FF is not set in Jetson orin Nano 

### ISSUE THREAD

**Issue by zainmfd** [kuriatsu/ros-g29-force-feedback#38]
I'm using  seeedstudio reComputer J4012-Edge AI Device with Jetson Orin NX 16GB module and trying g29 with ros noetic.
when I try to run os_g29_force_feedback it is giving an error like below 
![error](https://github.com/user-attachments/assets/136fb0a2-7318-4d5a-ad6b-1f6a5faff1dd)
Then I try  `zcat /proc/config.gz | grep CONFIG_LOGIWHEELS_FF`
Result
![Screenshot from 2024-09-12 21-42-22](https://github.com/user-attachments/assets/fad769c7-7802-4ed3-9cee-bfc7a9728295)
If anyone fix this issue please help me to solve this.

**Comment by kuriatsu** [comment]
It means your OS or Jetson doesn't support force feedback.
It seems nice to try installing Linux device drivers.
https://forums.developer.nvidia.com/t/logitech-g29-wheel-pedal-not-working/82078/3

**Comment by parthabana** [comment]
Hi @kuriatsu, thanks for the suggestion. I tried running your code and am getting the same error as @zainmfd got above. 
I am connecting the logitech g29 force-feedback steering wheel to a ubuntu 22.04 installed laptop. I have ros2 humble and am trying to run your node. I see that the device is connected on this event handler 
```
I: Bus=0003 Vendor=046d Product=c260 Version=0110
N: Name="Logitech G29 Driving Force Racing Wheel"
P: Phys=usb-0000:00:14.0-2.4/input0
S: Sysfs=/devices/pci0000:00/0000:00:14.0/usb1/1-2/1-2.4/1-2.4:1.0/0003:046D:C260.0001/input/input14
U: Uniq=
H: Handlers=event11 js0 
B: PROP=0
B: EV=1b
B: KEY=3fff000000000000 0 0 0 0
B: ABS=3003f
B: MSC=10
```
When running the node, I get the error as - 
```
ubuntu@d92ed2a42de4:~/workspace/pfr_ws$ ros2 run ros_g29_force_feedback g29_force_feedback --ros-args --params-file src/ros-g29-force-feedback/config/g29.yaml 
device opened
ERROR: force feedback is not supported
[ros2run]: Process exited with failure 1
```

The linux kernel version i have is - `6.8.0-51-generic`. I have the following installed - 
```
$ ls /boot/config-
config-5.15.0-100-generic  config-6.8.0-50-generic    
config-5.15.0-130-generic  config-6.8.0-51-generic  
```
When I try - `grep CONFIG_LOGIWHEELS_FF /boot/config-$(uname -r)`, I get this - `CONFIG_LOGIWHEELS_FF=y`. This means that the support for force-feedback on my current kernel version is enabled. 

Could you please help me identify what could be the problem? Happy to provide more details for debugging, if need be. Thanks!

**Comment by kuriatsu** [comment]
Did you change `device_name` in config/g29.yaml to `event11` ?

**Comment by parthabana** [comment]
Yes, I changed the device name parameter in the yaml file to device_name ='/dev/input/event11'. That's how the device was able to successfully connect. I think the issue is related to the force feedback modules. Could you please let me know if there were any other kernel versions that you have tested? Besides the 5.3.0-generic one?

**Comment by kuriatsu** [comment]
Sorry, I haven't checked the kernel and can't test it right now.
If `fftest /dev/input/event11` doesn't work, your kernel doesn't support force feedback on g29 now.
If it works, my program has a bug.

**Comment by kuriatsu** [comment]
If your wheel doesn't turn for calibration when you plug it in, there may be a hardware issue.

**Comment by parthabana** [comment]
I tried `fftest /dev/input/event11` and the wheel does not respond to any inputs. I suppose my kernel does not support force feedback. Also, there is no hardware issue as the wheel turns for calibration when I plug it in.

**Comment by kuriatsu** [comment]
Okay, that may be the kernel issue.
@zainmfd issue was that Jetson doesn't support FF (may be because of ARM).
I think you are using x86. If so, it's kinda weird that your PC doesn't support FF.
